app/src/main/java/com/github/satoshun/example/ci/MainActivity.kt

```diff
@@ -6,6 +6,6 @@ import android.support.v7.app.AppCompatActivity
 class MainActivity : AppCompatActivity() {
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-    setContentView(R.layout.main_act)
+     setContentView(R.layout.main_act)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/satoshun-example/AndroidCIExample/blob/1caa9462473f0ee4e98860fe1765c3f95dcabbb7/app/src/main/java/com/github/satoshun/example/ci/MainActivity.kt#L9"></span>Unexpected indentation (5) (it should be 4)</td>
    </tr>
  </tbody>
</table>